packages/core/src/utils/requestQueue.ts

```diff
@@ -154,6 +179,7 @@ export const createRequestQueue = ({
 function shouldRetry(error: Error) {
   if ("code" in error && typeof error.code === "number") {
     if (error.code === -1) return true; // Unknown error
+    if (error.code === InvalidInputRpcError.code) return true;
```

Comment:
Might be worth adding to the changeset.

Fixed a bug where certain RPC errors were incorrectly marked as non-retryable.